README.md

```diff
@@ -1,23 +1,26 @@
 ![](https://github.com/asineth0/checkn1x/blob/master/icon_dark.png?raw=true)
 
-# checkn1x
+# checkn1x Reborn - bring back Checkn1x with latest Checkra1n version (0.12.3)
 
 Linux-based distribution for jailbreaking iOS devices w/ checkra1n.
+Full latest changelogs at [here](https://checkra.in/releases/0.12.3-beta)
 
 ## Downloads
 
-Downloads are available under [releases](https://github.com/asineth0/checkn1x/releases).
+Downloads are available under [releases](https://github.com/TeGaX/checkn1x/releases).
 
 ## Usage
 
 **Use whatever tool you want, but I'm only officially supporting Etcher.**
 
-1. Download [Etcher](https://etcher.io) and the ISO from releases.
+1. Download [Etcher](https://etcher.io), [Rufus](https://rufus.ie) and the ISO from releases.
```

Comment:
Etcher works fine, Rufus is not needed. Also Rufus is windows only whereas etcher is cross platform.